src/game/boe.newgraph.cpp

```diff
@@ -120,6 +120,15 @@ rectangle explode_place_rect[30];
 
 char last_light_mask[13][13];
 
+static void terrain_screen_rects(rectangle& old_bounds, rectangle& new_bounds) {
+	rectangle rect = rectangle(terrain_screen_gworld);
+	rect.bottom -= 10; // Adjust for pointing buttons
```

Comment:
Wait… weren't the point buttons removed from the terrain screen sheet? We probably don't even need this line anymore?

Reply:
I got that it was the same thing, just not that it was the blue arrows.

Reply:
Those arrows used to be attached to the bottom of terscreen.png. They aren't anymore.